File: site/static/templates/blank_play_state/java/core/src/main/java/{{PACKAGE_PATH}}/{{GAME}}.java
package {{PACKAGE}};

import me.stringdotjar.flixelgdx.FlixelGame;

/**
 * Your game entry class.
 * <p>
 * FlixelGame owns the window settings and picks the first FlixelState.
 * <p>
 * If you are new here, start in PlayState.java. That file is where you spawn sprites,
 * load sounds, and write your first update loop.
 */
public final class {{GAME}} extends FlixelGame {

  public {{GAME}}() {
    super("{{GAME_NAME_ESC_JAVA}}", 640, 480, new PlayState());
  }
}

File: site/static/templates/blank_play_state/kotlin/core/src/main/kotlin/{{PACKAGE_PATH}}/{{GAME}}.kt
package {{PACKAGE}}

import me.stringdotjar.flixelgdx.FlixelGame

/**
 * Your game entry class.
 * <p>
 * FlixelGame owns the window settings and picks the first FlixelState.
 * <p>
 * If you are new here, start in PlayState.kt. That file is where you spawn sprites,
 * load sounds, and write your first update loop.
 */
class {{GAME}} : FlixelGame("{{GAME_NAME_ESC_KOTLIN}}", 640, 480, PlayState())
